README.md

```diff
@@ -816,7 +817,8 @@ cf push pora --no-start
 cf enable-service-access persi-nfs
 
 # Create a service and bind it
-cf create-service persi-nfs Existing myVolume -c '{"share":"192.168.77.77/exports/foo"}'
+EXTERNAL_IP=$(getent hosts "cf-dev.io" | awk 'NR=1{print $1}')
+cf create-service persi-nfs Existing myVolume -c "{\"share\":\"${EXTERNAL_IP}/exports/foo\"}"
```

Comment:
The line following this one uses `'` instead of escaping the `"`. Might be good to keep things consistent.

Reply:
I only changed this one because of the interpolation. I would prefer keeping how it is because the next line doesn't need any interpolation. It would just look more weird with the back-slashes:
```
cf bind-service pora myVolume -c "{\"uid\":\"1000\",\"gid\":\"1000\"}"
```

Reply:
:+1: